script.js

```diff
@@ -0,0 +1,40 @@
+const conteiner = document.querySelector(".conteiner");
+const leftArrow = document.querySelector(".left");
+const rightArrow = document.querySelector(".right");
+
+const quantityElemnts = parseInt(prompt("How much elements would you like to see?", "cancell for default"));
+if (typeof quantityElemnts === 'number' && !isNaN(quantityElemnts)) {
+  alert("IN!")
+  conteiner.style.width = quantityElemnts * 130 + "px";
+}
+
+const conteinerWidth = conteiner.offsetWidth;
+const lastSrcollPos = 1300 - conteinerWidth;
+
+visibilityToggle();
+
+leftArrow.addEventListener('click', handler);
+rightArrow.addEventListener('click', handler);
+
+function visibilityToggle () {
+  if(conteiner.scrollLeft === 0) {
+    leftArrow.classList.add('disableArrow');
```

Comment:
There is a convention to write CSS classes in `snake_case` or `cebab-case`, not in `camelCase`